winter/schema/path_parameters_inspector.py

```diff
@@ -16,14 +16,14 @@ def inspect_parameters(self, controller_method: ControllerMethod) -> List[openap
 
         parameters = []
 
-        for path_variable_name in uritemplate.variables(controller_method.url_path):
+        for path_variable_name in uritemplate.variables(controller_method.full_url_path):
```

Comment:
This solve issue-31

Reply:
Type it like this to automatically close the issue on PR merge: Closes #31